use-cases/personalized-video-messaging.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Personalized Video Messaging"
+description: "Create personalized video messages for your customers."
+---
+
+## Personalized Video Messaging
+
+Personalized video messaging involves creating unique video content tailored to individual recipients. This approach leverages customer data to dynamically generate videos, leading to higher engagement, better conversion rates, and a more personal connection compared to generic video messages. It's often used in sales outreach, marketing campaigns, and customer support.
+
+Follow these steps to run the personalized video messaging example:
+
+<Steps>
+  <Step title="Clone Github Repository">
+    Clone the [example github repository](https://github.com/synchronicity-labs/PersonalizedVideoMessaging).
```

Comment:
nit: i would create a new repo, f.ex: "sync-examples" and add bunch of examples in diff languages in subfolders. 

this one could be "sync-examples/personalized-video-messaging-py"

Reply:
done